src/customer-service/Actions/ShowSingleIncidentAction.cs

```diff
@@ -0,0 +1,85 @@
+﻿using customer_service.Models;
+using System;
+using System.Globalization;
+
+namespace customer_service
+{
+    /**
+     * Class: ShowSingleIncidentAction
+     * Purpose: Displays the menu and details for the single incident report
+     * Author: Dayne Wright
+     * Methods:
+     *     void ReadInput() - Shows the incident report and prompts for a resolution if one is not provided. 
+     *     Then saves that resolution with the current date.
+     */
+
+    public class ShowSingleIncidentAction
+    {
+        /**
+         * Purpose: Shows incident report and saves resolution if needed.
+         * Arguments:
+         *     NONE
+         * Return:
+         *     Void
+         */
+        public static void ReadInput()
+        {
+            Customer customer = CustomerFactory.Instance.ActiveCustomer;
+            Incident incident = IncidentFactory.Instance.ActiveIncident;
+
```

Comment:
I am not sure I understand.  The `incident` does not have a `CustomerId`.    We would have to make a method on the `IncidentFactory` that can  take an `incidentId` and return a `customer` .  I am not sure that is necessary (unless I am missing something) because the `customer` will be set by `Create Incident` or by the selection on `List Incidents` prior to getting to this screen for resolution on the `CustomerFactory.Instance.ActiveCustomer`.